actions/locations.js

```diff
@@ -2,7 +2,7 @@ import Sentry from 'sentry-expo';
 import * as apis from '../apis';
 import { requestPushPermission } from './permissions';
 import { containing } from './regions';
-import { failed, pending, uploaded } from './uploads';
+import { failed, pending, uploaded, offline, setUploadingStatus } from './uploads';
```

Comment:
What is the difference between offline and pending?

Reply:
Offline is what it sounds like, pending is when its currently being processed to upload. So in between offline and uploaded

Reply:
What is the benefit of offline vs pending.